cmd/apiserver/app/server/options.go

```diff
@@ -31,16 +31,16 @@ type ServiceCatalogServerOptions struct {
 	StorageTypeString string
 	// the runtime configuration of our server
 	GenericServerRunOptions *genericserveroptions.ServerRunOptions
+	// the admission options
+	AdmissionOptions *genericserveroptions.AdmissionOptions
 	// the https configuration. certs, etc
 	SecureServingOptions *genericserveroptions.SecureServingOptions
 	// authn for the API
 	AuthenticationOptions *genericserveroptions.DelegatingAuthenticationOptions
 	// authz for the API
 	AuthorizationOptions *genericserveroptions.DelegatingAuthorizationOptions
-	// InsecureOptions are options for serving insecurely.
-	InsecureServingOptions *genericserveroptions.ServingOptions
```

Comment:
Insecure (HTTP) server is not supported anymore, only HTTPS is supported by API server now